daphne/src/roles.rs

```diff
@@ -40,9 +41,12 @@ pub trait DapAuthorizedSender<S> {
 
 /// DAP Aggregator functionality.
 #[async_trait(?Send)]
-pub trait DapAggregator<'a, S>: HpkeDecrypter<'a> + Sized {
-    /// Return type of `get_task_config_for()`, wraps a reference to task configuration.
-    type WrappedDapTaskConfig: AsRef<DapTaskConfig>;
+pub trait DapAggregator<'srv, 'req, S>: HpkeDecrypter<'srv> + Sized
+where
+    'srv: 'req,
+{
+    /// A refernce to a task configuration stored by the Aggregator.
+    type WrappedDapTaskConfig: AsRef<DapTaskConfig> + 'req;
```

Comment:
Reviewer note: I don't think this bound is strictly necessary.
```suggestion
    type WrappedDapTaskConfig: AsRef<DapTaskConfig>;
```

Reply:
It does not. cargo seems to be able to infer that the lifetime of the output needs to live at least as long as `'req`. Interestingly, clippy will usually warn you about lifetimes that can be elided, but doesn't here.

Reply:
Removed.